src/routes/OrganizationsList/OrganizationsListTableData.tsx

```diff
@@ -0,0 +1,187 @@
+import {Td} from '@patternfly/react-table';
+import {Skeleton} from '@patternfly/react-core';
+import './css/Organizations.scss';
+import {Link} from 'react-router-dom';
+import {useEffect, useState} from 'react';
+import {fetchOrg} from 'src/resources/OrganizationResource';
+import {
+  fetchRepositoriesForNamespace,
+  IRepository,
+} from 'src/resources/RepositoryResource';
+import {fetchMembersForOrg} from 'src/resources/MembersResource';
+import {fetchRobotsForNamespace} from 'src/resources/RobotsResource';
+import {formatDate} from 'src/libs/utils';
+import ColumnNames from './ColumnNames';
+
+type TeamCountProps = {
+  orgName: string;
+  user: boolean;
+};
+
+type MemberCountProps = {
+  orgName: string;
+  user: boolean;
+};
+
+type RobotCountProps = {
+  orgName: string;
+};
+
+type RepoCountProps = {
+  count: string | number;
+};
+
+type OrgTableProps = {
+  orgName: string;
+  user: boolean;
+};
+
+type RepoLastModifiedDateProps = {
+  lastModifiedDate: number;
+};
+
+function TeamCount(props: TeamCountProps) {
+  const [teamCount, setTeamCount] = useState<string | number>(null);
+
+  useEffect(() => {
+    let count = null;
+    if (!props.user) {
+      fetchOrg(props.orgName)
+        .then((data) => {
+          count = data?.teams ? Object.keys(data?.teams)?.length : 0;
+          setTeamCount(count);
+        })
+        .catch((err) => {
+          console.error(err);
+          count = 'N/A';
+          setTeamCount(count);
+        });
+    } else {
+      count = 'N/A';
+      setTeamCount(count);
+    }
+  }, [props.orgName]);
+
+  return <>{teamCount !== null ? teamCount : <Skeleton width="100%" />}</>;
+}
+
+function MemberCount(props: MemberCountProps) {
+  const [memberCount, setMemberCount] = useState<string | number>(null);
+
+  useEffect(() => {
+    let count = null;
+    if (!props.user) {
+      fetchMembersForOrg(props.orgName)
+        .then((data) => {
+          count = data.length;
+          setMemberCount(count);
+        })
+        .catch((err) => {
+          console.error(err);
+          count = 'N/A';
+          setMemberCount(count);
+        });
+    } else {
+      count = 'N/A';
+      setMemberCount(count);
+    }
+  }, [props.orgName]);
```

Comment:
I don't think we'll need to re-fire based on the `orgName` since an entirely new component is rendered when ever a table row is rendered with a different org name. This may lead to unintended behavior.